.pre-commit-config.yaml

```diff
@@ -1,20 +1,29 @@
-# TODO: add more hooks from https://pre-commit.com/hooks.html
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.3.0
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: check-added-large-files
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 22.6.0
     hooks:
       - id: black
   - repo: https://github.com/PyCQA/isort
     rev: 5.10.1
     hooks:
       - id: isort
+  - repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - flake8-black>=0.3.3
+        args: ['--config=.flake8']
   - repo: https://github.com/clairBuoyant/pre-commit
     rev: v0.1.1
     hooks:
       - id: branch-validate
+# TODO:
+# add commitizen: https://github.com/commitizen-tools/commitizen
+# add formatter for bash: https://github.com/lovesegfault/beautysh
```

Comment:
What do you think about adding YAML link hook?

Reply:
lint*

Reply:
Thanks Jake! That's a great idea. I made a note to incorporate it if/when we need to add more yaml files.

Since we only have one yaml file, I don't think we need to run a hook for it at the moment.